test/runtests.jl

```diff
@@ -523,42 +526,46 @@ end
     L = LenseFlow{RK4Solver{7}}
     T = Float64
     
-    for pol in (:I,:P)
-        
-        @testset "pol = $pol" begin
-            
-            @unpack f,f̃,ϕ,ds,ds₀ = load_sim(
-                seed  = 0,
-                Cℓ    = Cℓ,
-                θpix  = 3,
-                Nside = 128,
-                T     = T,
-                beamFWHM = 3,
-                pol   = pol,
-                L     = L,
-                pixel_mask_kwargs = (edge_padding_deg=2,)
-                );
-            @unpack Cf,Cϕ,D = ds₀
-            f° = L(ϕ)*D*f
-
-            @test lnP(0,f,ϕ,ds) ≈ lnP(1,    f̃,  ϕ ,ds) rtol=1e-4
-            @test lnP(0,f,ϕ,ds) ≈ lnP(:mix, f°, ϕ, ds) rtol=1e-4
-
-            ε = sqrt(eps(T))
-            seed!(0)
-            δf,δϕ = simulate(Cf),simulate(Cϕ)
+
+    for nside = [128, (128, 64), (64, 128)]
```

Comment:
For all the new `for` loops, can you add a testset so it prints out on which `nside` it failed (when/if it does?). I think the syntax is something like

```julia
@testset "Nside $nside" for nside = [128, (128, 64), (64, 128)]
```

(You may want to double check / see https://docs.julialang.org/en/v1/stdlib/Test/ I haven't tried running that)

Reply:
Yup. Done.